dcmgr/lib/dcmgr/models/instance.rb

```diff
@@ -37,6 +37,7 @@ class Instance < AccountResource
     subset(:runnings, {:state => STATE_RUNNING})
     subset(:running_or_initializing, {:state => [STATE_RUNNING, STATE_INITIALIZING]})
     subset(:stops, {:state => STATE_STOPPED})
+    subset(:expired) { password_will_be_deleted_at <= Time.now.utc }
```

Comment:
I think the word `expired` isn't a clear description. How about `:password_deletion_time_passed`? It's long but more descriptive.